applications/source/ftp-source/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>ftp-source</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+    <name>ftp-source</name>
+    <description>ftp source apps</description>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>org.springframework.cloud.stream.app</groupId>
+        <artifactId>stream-applications-core</artifactId>
+        <version>3.0.0-SNAPSHOT</version>
```

Comment:
That's a good point. Can we address this later though? (I mean definitely before the first milestone release). We need to think through the versions a bit further since this repository maintains multiple version hierarchies.